scripts/scripts.js

```diff
@@ -75,10 +112,14 @@ export function decorateMain(main) {
  */
 async function loadEager(doc) {
   document.documentElement.lang = 'en';
-  decorateTemplateAndTheme();
+  const templateName = getMetadata('template');
```

Comment:
Not sure if we need templating at this stage given we have 1 page and its styles can be global. wdyt?

Reply:
done removed template usage